### DISCUSSION TITLE
Unpin Github Repository that I don't own

### DISCUSSION THREAD

**Discussion by heliohachimine** [community/community#86723] [New to GitHub]
Hello Github
I recently was cleaning up my Github profile and wanted to unpin one of the repositories that was marked on my profile. For some reason when I click on Customize my Pins, Save Pins button is greyed out no matter what I choose. In addition, when I click into the repository, I am not able to see the Pin / Unpin button that should be to the right of Watch. Because of that, I can't unpin the repository. I attached some photos of the problem childs.

**Comment by bonizario** [comment]
This issue might be due to a temporary glitch in GitHub's interface or a browser issue. Here are a few things you can try:

1. **Clear your browser cache and cookies**: Sometimes, old data stored in your browser can cause issues with web pages. Clearing your browser cache and cookies can often resolve these issues.

2. **Try a different browser or incognito mode**: There might be an issue specific to the browser you're using. Trying a different browser or using incognito mode can help determine if this is the case.

3. **Check for browser extensions that might be interfering**: Some browser extensions can interfere with web pages, especially ones that modify the page's JavaScript or CSS. Try disabling extensions to see if this resolves the issue.

4. **Wait and try again later**: If none of the above solutions work, it might be a temporary issue on GitHub's end. In this case, waiting a while and trying again later might resolve the issue.

If none of these solutions work, you might want to contact GitHub support for further assistance.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
